Modules/_testcapimodule.c

```diff
@@ -7304,6 +7304,7 @@ PyInit__testcapi(void)
     PyModule_AddObject(m, "PY_SSIZE_T_MAX", PyLong_FromSsize_t(PY_SSIZE_T_MAX));
     PyModule_AddObject(m, "PY_SSIZE_T_MIN", PyLong_FromSsize_t(PY_SSIZE_T_MIN));
     PyModule_AddObject(m, "SIZEOF_TIME_T", PyLong_FromSsize_t(sizeof(time_t)));
+    PyModule_AddObject(m, "Py_Version", PyLong_FromLong(Py_Version));
```

Comment:
It's no longer a long, but an unsigned long:

```suggestion
    PyModule_AddObject(m, "Py_Version", PyLong_FromUnsignedLong(Py_Version));
```